ts/features/itwallet/common/components/ItwSkeumorphicCard/index.tsx

```diff
@@ -1,31 +1,83 @@
-import React from "react";
-import { StyleSheet } from "react-native";
-import { StoredCredential } from "../../utils/itwTypesUtils";
+import React, { ReactNode, useMemo } from "react";
+import { StyleProp, StyleSheet, View, ViewStyle } from "react-native";
+import { Tag } from "@pagopa/io-app-design-system";
+import {
+  ItwCredentialStatus,
+  StoredCredential
+} from "../../utils/itwTypesUtils";
+import {
+  borderColorByStatus,
+  tagPropsByStatus,
+  validCredentialStatuses
+} from "../../utils/itwCredentialUtils";
 import { CardBackground } from "./CardBackground";
 import { CardData } from "./CardData";
 import { FlippableCard } from "./FlippableCard";
 
+type CardSideBaseProps = {
+  status: ItwCredentialStatus;
+  children: ReactNode;
+};
+
+const CardSideBase = ({ status, children }: CardSideBaseProps) => {
+  const statusTagProps = tagPropsByStatus[status];
+  const borderColor = borderColorByStatus[status];
+
+  const dynamicStyle: StyleProp<ViewStyle> = {
+    borderColor,
+    backgroundColor: validCredentialStatuses.includes(status)
+      ? undefined
+      : "rgba(255,255,255,0.7)"
+  };
+
+  return (
+    <View>
+      {statusTagProps && (
+        <View style={styles.tag}>
+          <Tag {...statusTagProps} />
+        </View>
+      )}
+      <View style={[styles.faded, dynamicStyle]} />
+      {children}
+    </View>
+  );
+};
+
 export type ItwSkeumorphicCardProps = {
   credential: StoredCredential;
+  status?: ItwCredentialStatus;
```

Comment:
This type allows for inconsistencies. Since we already have `StoredCredential` in the props, we could use the cached value returned by `itwCredentialStatusSelector` directly within the component. This approach would reduce the risk of discrepancies. What do you think?

Reply:
It makes sense. Addressed in 8d47b9b78ca031255f913aaad60f3e580e18b5ee.